engine/classes/Elgg/Comments/Preloader.php

```diff
@@ -114,7 +114,7 @@ public static function preload(\Elgg\Hook $hook): void {
 		if (!$preload) {
 			return;
 		}
-		
```

Comment:
shame, if this is the only change please undo this to the file doesn't get touched